_maps/map_files/Delta/delta.dmm

```diff
@@ -76358,17 +76358,17 @@
 /turf/simulated/floor/plasteel/dark,
 /area/station/engineering/smes)
 "fRb" = (
-/obj/structure/plasticflaps{
-	opacity = 1
-	},
 /obj/effect/turf_decal/delivery,
 /obj/machinery/door/window/classic/reversed{
 	dir = 8;
 	name = "Glass Door"
 	},
-/obj/effect/mapping_helpers/airlock/windoor/access/any/supply/mule_bot{
+/obj/effect/mapping_helpers/airlock/windoor/access/all/supply/general{
 	dir = 8
 	},
```

Comment:
Ideally, we'd be more explicit with the access, rather than generalizing it. I think you should have 2 anys here instead
```suggestion
/obj/effect/mapping_helpers/airlock/windoor/access/any/supply/mule_bot{
	dir = 8
	},
/obj/effect/mapping_helpers/airlock/windoor/access/any/supply/mail_sorting{
	dir = 8
	},
```

Reply:
so remove general supply access and put instead mule_bot and mail_sorting access instead?

Reply:
Yeah, but make sure theyre both any though

Reply:
I set it for this 
![Screenshot 2023-12-14 104054](https://github.com/ParadiseSS13/Paradise/assets/98861947/e438ea27-5709-4ac1-8ce2-10204b3a3ba6)